test-engine-core/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="test_engine_core",
-    version="0.9.0",
+    version="0.10.0",
```

Comment:
I think this version can remain at 0.9.0, most likely versioning is done in another way?